datatable/frame.py

```diff
@@ -563,8 +563,7 @@ def _delete_columns(self, cols):
             newnames += self.names[(cols[i - 1] + 1):cols[i]]
         newnames += self.names[cols[-1] + 1:]
         self._fill_from_dt(self._dt, names=newnames)
-
-
+        
```

Comment:
Looks like your editor doesn't have "Trim trailing whitespace" option turned on.

Reply:
Done. Actually C++ editor had this option turned on, had to fix the Python one only.